[2023-09-10T18:14:12.473+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: weather_pipeline.check_api_available manual__2023-09-10T18:14:10.810295+00:00 [queued]>
[2023-09-10T18:14:12.483+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: weather_pipeline.check_api_available manual__2023-09-10T18:14:10.810295+00:00 [queued]>
[2023-09-10T18:14:12.483+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-09-10T18:14:12.483+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-09-10T18:14:12.483+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-09-10T18:14:12.498+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): check_api_available> on 2023-09-10 18:14:10.810295+00:00
[2023-09-10T18:14:12.506+0000] {standard_task_runner.py:52} INFO - Started process 1300 to run task
[2023-09-10T18:14:12.510+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline', 'check_api_available', 'manual__2023-09-10T18:14:10.810295+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpipzx8gu_', '--error-file', '/tmp/tmpalkql7a0']
[2023-09-10T18:14:12.512+0000] {standard_task_runner.py:80} INFO - Job 47: Subtask check_api_available
[2023-09-10T18:14:12.588+0000] {task_command.py:371} INFO - Running <TaskInstance: weather_pipeline.check_api_available manual__2023-09-10T18:14:10.810295+00:00 [running]> on host afe1e7d20f24
[2023-09-10T18:14:12.674+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=thinh pham
AIRFLOW_CTX_DAG_ID=weather_pipeline
AIRFLOW_CTX_TASK_ID=check_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-09-10T18:14:10.810295+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-10T18:14:10.810295+00:00
[2023-09-10T18:14:12.689+0000] {logging_mixin.py:115} INFO - API request was successful!
[2023-09-10T18:14:12.689+0000] {logging_mixin.py:115} INFO - Response:
[2023-09-10T18:14:12.689+0000] {logging_mixin.py:115} INFO - {"coord":{"lon":-0.1257,"lat":51.5085},"weather":[{"id":804,"main":"Clouds","description":"overcast clouds","icon":"04d"}],"base":"stations","main":{"temp":297.89,"feels_like":298.04,"temp_min":295.07,"temp_max":300.25,"pressure":1012,"humidity":62},"visibility":10000,"wind":{"speed":4.12,"deg":250},"clouds":{"all":85},"dt":1694369563,"sys":{"type":2,"id":2075535,"country":"GB","sunrise":1694323612,"sunset":1694370526},"timezone":3600,"id":2643743,"name":"London","cod":200}
[2023-09-10T18:14:12.689+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-09-10T18:14:12.705+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=weather_pipeline, task_id=check_api_available, execution_date=20230910T181410, start_date=20230910T181412, end_date=20230910T181412
[2023-09-10T18:14:12.761+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-09-10T18:14:12.812+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
